op-chain-ops/genesis/helpers.go

```diff
@@ -18,13 +18,13 @@ var (
 	codeNamespace = common.HexToAddress("0xc0D3C0d3C0d3C0D3c0d3C0d3c0D3C0d3c0d30000")
 	// l2PredeployNamespace represents the namespace of L2 predeploys
 	l2PredeployNamespace = common.HexToAddress("0x4200000000000000000000000000000000000000")
-	// bigL2PredeployNamespace represents the predeploy namespace as a big.Int
+	// BigL2PredeployNamespace represents the predeploy namespace as a big.Int
 	BigL2PredeployNamespace = new(big.Int).SetBytes(l2PredeployNamespace.Bytes())
-	// bigCodeNamespace represents the predeploy namespace as a big.Int
+	// bigCodeNameSpace represents the predeploy namespace as a big.Int
 	bigCodeNameSpace = new(big.Int).SetBytes(codeNamespace.Bytes())
```

Comment:
nit: I would change the var name here instead of the comment (everything else uses Namespace with lowercase 's')

Reply:
There is some tech debt here - we used to deploy the L1 contracts as predeploys for the devnet, things are a bit over abstracted now that we only have L2 predeploys again